src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/StringSearchValue.cs

```diff
@@ -53,6 +62,23 @@ public void AcceptVisitor(ISearchValueVisitor visitor)
             visitor.Visit(this);
         }
 
+        /// <inheritdoc />
+        public int CompareTo(ISupportSortSearchValue supportSortSearchValue, ComparisonRange range)
+        {
+            if (supportSortSearchValue == null)
+            {
+                throw new ArgumentException("Value to be compared to cannot be null");
+            }
+
+            var otherValue = supportSortSearchValue as StringSearchValue;
+            if (otherValue == null)
+            {
+                throw new ArgumentException($"Value to be compared should be of type {typeof(StringSearchValue)}");
+            }
+
+            return string.Compare(ToString(), otherValue.ToString(), CultureInfo.InvariantCulture, CompareOptions.IgnoreNonSpace | CompareOptions.IgnoreCase);
```

Comment:
@johnstairs any thoughts here? Ideally this needs to match how the database sorts / deals with accents.
//cc @namadabu

Reply:
Turns out we are not properly normalizing in [Cosmos DB](https://github.com/Microsoft/fhir-server/blob/268834e6fbdc6e7d167672a44adb6ef16e122902/src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Search/SearchIndexEntryJObjectGenerator.cs#L123-L134). We're not removeing accents. So that's a bug.
The collation we are using is SQL would be roughly equivalent to what you have here.

Reply:
Yes, this bug has been raised. Since sort will be serializing to a new field, we could make it consistent for this